### DISCUSSION TITLE
Mi Práctica CSS

### DISCUSSION THREAD

**Discussion by RamSantoyo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#315] [Show and tell]
https://practical-gates-267e66.netlify.app/
Por ahora solo esta para escritorio

**Comment by sergiolopezloya** [comment]
Se va viendo muy bien compañero :)

**Comment by JuanSalvador73** [comment]
Va muy bien. ¿Usaste una resolución de 1920 x 1080?

**Reply by RamSantoyo** [reply]
1440 x 1363 bro
